app/jobs/moderators/bust_cache_job.rb

```diff
@@ -0,0 +1,12 @@
+module Moderators
+  class BustCacheJob < ApplicationJob
+    queue_as :moderators_bust_cache
+
+    def perform(user_id, cache_buster = CacheBuster)
```

Comment:
In my opinion, this job is too specific. Currently, it's only usable for busting only one specific path.
What do you think about having a more universal job, e.g. passing a path as an argument and busting it? There will be no need to find a user too, in case we want to use this job after user deletion. The more universal job could also be used to bust others' paths caches asyncronously.